### PR TITLE
Added creation date and last updated date labels

### PR DESCRIPTION
**Pull request by sanyavertolet**
### What's done:
 * Added creation date and last updated date to `VulnerabilityDto
 * Added creation date to `fossGraph` view
 * Added last updated date to `fossGraph` view

### How it looks:

screenshot

### CODE REVIEW

**Review by nulls — APPROVED** [review]

File: save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entities/vulnerabilities/Vulnerability.kt [hunk 1]
```diff
@@ -97,5 +101,7 @@ class Vulnerability(
         dates = emptyList(),
         participants = emptyList(),
         status = status,
+        creationDateTime = createDate?.toKotlinLocalDateTime(),
```
Comment (nulls) [inline save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entities/vulnerabilities/Vulnerability.kt:104]: can we change field name in entity and database to `createdDate`\`created` and `updatedDate`\`lastUpdatedDate`\`lastUpdated` accordingly ?

**Review by akuleshov7 — APPROVED** [review]
lgtm, quick solution

File: save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entities/vulnerabilities/Vulnerability.kt (hunk 1 above)
Comment (Cheshiriks) [reply save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entities/vulnerabilities/Vulnerability.kt:104]: We will have to change a lot of tables. Update and Create Date are BaseEntity fields.

**Review by Cheshiriks — APPROVED** [review]

File: save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entities/vulnerabilities/Vulnerability.kt (hunk 1 above)
Comment (nulls) [reply save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entities/vulnerabilities/Vulnerability.kt:104]: I see: `com.saveourtool.save.spring.entity.BaseEntityWithDate` will create an issue in backlog for it

Comment (sanyavertolet) [reply save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entities/vulnerabilities/Vulnerability.kt:104]: > We will have to change a lot of tables. Update and Create Date are BaseEntity fields.

©Cheshiriks

**Pull request merged**